components/schemas/scoped-variables/URLSource.yml

```diff
@@ -15,14 +15,16 @@ properties:
     required:
       - url
       - headers
+      - auth_token_url
     properties:
       url:
         type: string
         description: The URL to call to produce the value.
       headers:
         type: object
-        description: Additional information about a container
+        description: Additional information about a URL source
```

Comment:
punctuation, but try to bring this inline with the other description for the source block.

Reply:
is that what headers is? that doesn't seem right.